domain/src/main/java/org/oppia/android/domain/platformparameter/syncup/PlatformParameterSyncUpWorkManagerInitializer.kt

```diff
@@ -0,0 +1,76 @@
+package org.oppia.android.domain.platformparameter.syncup
+
+import android.annotation.SuppressLint
+import android.content.Context
+import androidx.work.Constraints
+import androidx.work.Data
+import androidx.work.ExistingPeriodicWorkPolicy
+import androidx.work.NetworkType
+import androidx.work.PeriodicWorkRequest
+import androidx.work.WorkManager
+import org.oppia.android.domain.oppialogger.ApplicationStartupListener
+import org.oppia.android.util.platformparameter.PlatformParameterValue
+import org.oppia.android.util.platformparameter.SyncUpWorkerTimePeriod
+import java.util.UUID
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Enqueues unique periodic work requests for fetching and caching latest platform parameter values
+ * from the remote service on application creation.
+ */
+@Singleton
```

Comment:
Is this injected in more than one place? Since it doesn't contain state, we don't actually need to scope it.

Reply:
No its not injected more than once. Done